doc/content/source/userobjects/OpenMCDomainFilterEditor.md

```diff
@@ -0,0 +1,20 @@
+# OpenMCDomainFilterEditor
+
+!syntax description /UserObjects/OpenMCDomainFilterEditor
+
+## Description
+
+This user object can be used to create or control new domain filter in OpenMC.
+Currently suppored filter types are "cell", "material", "universe", or "mesh"
```

Comment:
In the interest of technical debt, I'd revise this sentence to "Currently supported filter types are specified with the 'filter_type' parameter."

Reply:
Good call! I'll use this.